__tests__/podlet.js

```diff
@@ -254,6 +273,44 @@ test('Podlet() - serialize default values - should set "proxy" to empty Object',
     expect(result.proxy).toEqual({});
 });
 
+test('Podlet() - should collect metric with version info', done => {
+    jest.useFakeTimers();
```

Comment:
Related to my second comment; but if the purpose of the timeout around the metric producer is to run stuff on the next tick, we should not need to fake timers.